changelog.d/issue-5993

```diff
@@ -0,0 +1,4 @@
+synopsis: Warn early that overwrite policy is needed
```

Comment:
Currently it sounds a bit cryptic and rough. This could be expanded to explain the issue a bit better. Either right in `sinopsis` and/or in `description`.

Reply:
Thanks for suggesting this. I added a description.

Reply:
Thanks!